x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformIndexer.java

```diff
@@ -646,12 +649,6 @@ protected void onFailure(Exception exc) {
         }
     }
 
-    @Override
-    protected void onStop() {
-        auditor.info(transformConfig.getId(), "Transform has stopped.");
-        logger.info("[{}] transform has stopped.", transformConfig.getId());
```

Comment:
Maybe it's still worth logging _something_ here so we can detect the situation of something hanging in between this and `afterFinishOrFailure`?

I agree we shouldn't audit two notifications, but maybe just a log message could be helpful and it shouldn't be _that_ noisy.

Reply:
:+1: I added the log back as `debug`.